server/index.js

```diff
@@ -2,43 +2,101 @@
 /* eslint-disable no-unused-vars */
 /* eslint-disable no-console */
 const express = require('express');
+
+const app = express();
 const axios = require('axios');
 const utils = require('../database/utils.js');
 const db = require('../database/index.js');
 
-const app = express();
 const port = 3004;
 
 app.use('/items/:itemId', express.static('client/dist'));
 app.use(express.json());
-app.use(express.urlencoded());
+app.use(express.urlencoded({ extended: false }));
 
-// Get all item data from db
+// Get all items from db
 app.get('/shopping/items', (req, res) => {
-  console.log('GET request for all items successful');
-  db
-    .query('SELECT * FROM items')
-    .then((result) => {
-      console.log(result.rows);
-      res.send(result.rows);
-    })
-    .catch((error) => {
-      console.error(error);
-      res.send('Error getting data from db');
-    });
+  db.model.getAll((err, data) => {
```

Comment:
Great start to MVC, model looks good. Might want to look into a controller folder/file - let me know if you want help. 
Server --> Controller --> Model

Reply:
MVC is my next step goal, but MVP was my short term goal :). I'll most likely have questions for you about this.